### PR TITLE
Fix w2grid getCellHTML title

### PR DESCRIPTION
**Pull request by mpf82**
Make title behave consistently accross the various renderers.

### CODE REVIEW

**Comment by mpf82** [comment]
In case someone wants to write a unit test:

```
<div id="grid" style="height: 300px;"></div>

function test_grid() {
    var render_obj                = {
        "Hannibal the ..." : "Hannibal the ... object",
    };
    var render_func               = ( record, ind, col_ind, data ) => data + " func";
    w2utils.formatters.render_str = ( data, params, record ) => data + " str";
    var render_title              = data => "My custom title";
    $().w2destroy( "grid" );
    $( "#grid" ).w2grid( {
        name    : "grid",
        columns : [
            { field : "name", caption : "render:null", render : null },
            { field : "name", caption : "render:obj", render : render_obj },
            { field : "name", caption : "render:func", render : render_func },
            { field : "name", caption : "render:str", render : "render_str" },
            // ----
            { field : "name", caption : "render:null NT", render : null, title : "" },
            { field : "name", caption : "render:obj NT", render : render_obj, title : "" },
            { field : "name", caption : "render:func NT", render : render_func, title : "" },
            { field : "name", caption : "render:str NT", render : "render_str", title : "" },
            // ----
            { field : "name", caption : "render:null TF", render : null, title : render_title },
            { field : "name", caption : "render:obj TF", render : render_obj, title : render_title },
            { field : "name", caption : "render:func TF", render : render_func, title : render_title },
            { field : "name", caption : "render:str TF", render : "render_str", title : render_title },
        ],
        records : [
            {
                recid : 1,
                name  : "Hannibal the ...",
            },
        ],
        total : 1,
    } );
}
test_grid();
```

File: src/w2grid.js [hunk 1]
```diff
@@ -7478,9 +7479,21 @@
                     '></span>';
             }
             // various renderers
+            function getTitle(cellData){
```
Comment (vitmalina) [inline src/w2grid.js:7482]: The only suggestion is to move this function to the end of the parent function. I think it is more readble to have nested functions at the end. Even after `return`. I will do it.

File: src/w2grid.js [hunk 2]
```diff
@@ -7489,14 +7502,15 @@
...
-                        data = '<div style="'+ style +'">' + infoBubble + String(data) + '</div>';
+                        data = '<div style="'+ style +'" title="'+ getTitle(data) +'">' + infoBubble + String(data) + '</div>';
                     }
                 }
                 // if it is an object
                 if (typeof col.render == 'object') {
                     var dsp = col.render[data];
                     if (dsp == null || dsp === '') dsp = data;
-                    data = '<div style="'+ style +'">' + infoBubble + dsp + '</div>';
+                    data = dsp;
```
Comment (vitmalina) [inline src/w2grid.js:7512]: this line is confusing, as in the next line data is set again

**Review by vitmalina — APPROVED** [review]

File: src/w2grid.js (hunk 2 above)
Comment (mpf82) [reply src/w2grid.js:7512]: true, I only did that so the line ``data = '<div style="'+ style +'" title="'+ getTitle(data) +'">' + infoBubble + String(data) + '</div>';``  is the same for all cases

**Pull request merged**